src/main/java/io/vertx/core/shareddata/AsyncMap.java

```diff
@@ -108,7 +109,7 @@
    * @param v  the new value
    * @param resultHandler  the result handler will be passed the previous value
    */
-  void replace(K k, V v, Handler<AsyncResult<V>> resultHandler);
+  void replace(K k, V v, Handler<AsyncResult<@Nullable V>> resultHandler);
```

Comment:
the second param `V` should also be `@Nullable`

Reply:
No, null values are forbidden in AsyncMap.

Reply:
ah ok makese sense then, perhaps we should say it in the javadoc as well then